pytensor/link/pytorch/dispatch/extra_ops.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from pytensor.link.pytorch.dispatch.basic import pytorch_funcify
+from pytensor.tensor.extra_ops import CumOp
+
+
+@pytorch_funcify.register(CumOp)
+def pytorch_funcify_Cumop(op, **kwargs):
+    dim = op.axis
+    mode = op.mode
+
+    def cumop(x, dim=dim, mode=mode):
```

Comment:
This is not needed, the returned functions are never called by the user
```suggestion
    def cumop(x):
```